### PR TITLE
Fixing a small bug

### PR DESCRIPTION
**Pull request by ewjoachim**

### CODE REVIEW

File: c.html [hunk 1]
```diff
@@ -62,12 +62,12 @@
 	var validPassport = 0
 
 	// Regex
-	var regex4digits = new RegExp("([0-9]{4})$")
-	var regexHeight = new RegExp("([0-9][A-Za-z]{2})$")
-	var regexHexaColor = new RegExp("^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$")
-	var regexLeadingZeros = new RegExp(/[0-9]{4,100}$/);
+	var regex4digits = new RegExp("^([0-9]{4})$")
```
Comment (ewjoachim) [inline c.html:65]: Adding a `^` at the begnning, otherwise `.test()` will return `true` even if the pattern starts incorrectly.

File: c.html [hunk 2]
```diff
@@ -62,12 +62,12 @@
...
+	var regexHeight = new RegExp("^([0-9]+[A-Za-z]{2})$")
```
Comment (ewjoachim) [inline c.html:66]: Here, I'm adding a `^` and adding a `+` after the digit, because otherwise only a single digit is matched.

File: c.html [hunk 3]
```diff
@@ -62,12 +62,12 @@
...
+	var regexHexaColor = new RegExp("^#([A-Fa-f0-9]{6})$")
```
Comment (ewjoachim) [inline c.html:67]: The specifications does not allow for a 3 letter hex color (even if this would be a valid CSS color)

File: c.html [hunk 4]
```diff
@@ -62,12 +62,12 @@
...
+	var regexLeadingZeros = new RegExp(/^[0-9]{9}$/);
```
Comment (ewjoachim) [inline c.html:68]: The specifications say the number must be 9 digits exactly

**Review by Clotilde — COMMENTED** [review]
Merci @ewjoachim !

**Pull request closed without merging**